### PR TITLE
Update CrystalConf 2023 into retrospective + blog post

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

File: _posts/2023-11-09-memories-from-crystalconf.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<a href="https://youtube.com/playlist?list=PLt-CsM4G1WoadONHl3zPN_Ts5PqH8TgMZ" class="btn btn-large btn-flat">Video playlist</a>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/audience.jpg)
+  <caption>The audience listens to Johannes' talk about noticing little details that point towards deficiencies.</caption>
+</figure>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/panel.jpg)
+  <caption>In the panel discussion we discussed the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>
```
Comment (matiasgarciaisaia) [inline _posts/2023-11-09-memories-from-crystalconf.md:21]:
```suggestion
  <caption>In the panel discussion we talked the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>
```

File: _posts/2023-11-09-memories-from-crystalconf.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Memories from CrystalConf
+author: straight-shoota
+summary: Impressions from the conference in Berlin
+---
+
+CrystalConf 2023 was a sounding success with a terrific lineup of speakers and talks.
+We spent two great days in Berlin sharing knowledge, ideas, and beers.
+
+Recordings of the talks are available on YouTube.
```
Comment (beta-ziliani) [inline _posts/2023-11-09-memories-from-crystalconf.md:10]: If we want to send this soon, we should mention the list is being constructed.

```suggestion
Recordings of the talks are being uploaded to a playlist on YouTube.
```

File: _posts/2023-11-09-memories-from-crystalconf.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<a href="https://youtube.com/playlist?list=PLt-CsM4G1WoadONHl3zPN_Ts5PqH8TgMZ" class="btn btn-large btn-flat">Video playlist</a>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/audience.jpg)
+  <caption>The audience listens to Johannes' talk about noticing little details that point towards deficiencies.</caption>
```
Comment (beta-ziliani) [inline _posts/2023-11-09-memories-from-crystalconf.md:16]: Is it just me? The many indirections makes this hard to parse.

```suggestion
  <caption>The audience listens to Johannes' talk about deficiencies in programs that might come from hidden little details.</caption>
```

File: _posts/2023-11-09-memories-from-crystalconf.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+</figure>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/panel.jpg)
+  <caption>In the panel discussion we talked the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>
```
Comment (beta-ziliani) [inline _posts/2023-11-09-memories-from-crystalconf.md:21]: Do we mention anything of Stephie being back?

Comment (straight-shoota) [reply _posts/2023-11-09-memories-from-crystalconf.md]: I simply shortened the caption.

**Review by beta-ziliani — APPROVED** [review]
